spec/helpers/make_helpers.rb

```diff
@@ -202,31 +202,33 @@ def make_threatened_taxon( options = {} )
-    set_taxon_with_rank_and_parent( "Pseudacris", Taxon::GENUS, @Hylidae )
-    set_taxon_with_rank_and_parent( "Pseudacris regilla", Taxon::SPECIES, @Pseudacris )
+    ActiveRecord::Base.transaction do
+      @Life = Taxon.find_by_name( "Life" ) || Taxon.make!( name: 'Life', rank: "state of matter" )
 
-    set_taxon_with_rank_and_parent( "Aves", Taxon::CLASS, @Chordata, is_iconic: true )
-    set_taxon_with_rank_and_parent( "Apodiformes", Taxon::ORDER, @Aves )
-    set_taxon_with_rank_and_parent( "Trochilidae", Taxon::FAMILY, @Apodiformes )
-    set_taxon_with_rank_and_parent( "Calypte", Taxon::GENUS, @Trochilidae )
-    set_taxon_with_rank_and_parent( "Calypte anna", Taxon::SPECIES, @Calypte, common_name: "Anna's Hummingbird" )
-    
-    set_taxon_with_rank_and_parent( "Plantae", Taxon::KINGDOM, @Life, is_iconic: true )
-    set_taxon_with_rank_and_parent( "Magnoliophyta", Taxon::PHYLUM, @Plantae )
-    set_taxon_with_rank_and_parent( "Magnoliopsida", Taxon::CLASS, @Magnoliophyta )
-    set_taxon_with_rank_and_parent( "Myrtales", Taxon::ORDER, @Magnoliopsida )
-    set_taxon_with_rank_and_parent( "Onagraceae", Taxon::FAMILY, @Myrtales )
-    set_taxon_with_rank_and_parent( "Clarkia", Taxon::GENUS, @Onagraceae )
-    set_taxon_with_rank_and_parent( "Clarkia amoena", Taxon::GENUS, @Clarkia )
+      set_taxon_with_rank_and_parent( "Animalia", Taxon::KINGDOM, @Life, is_iconic: true )
+      set_taxon_with_rank_and_parent( "Chordata", Taxon::PHYLUM, @Animalia )
+      set_taxon_with_rank_and_parent( "Amphibia", Taxon::CLASS, @Chordata, is_iconic: true )
+      set_taxon_with_rank_and_parent( "Anura", Taxon::ORDER, @Amphibia )
+      set_taxon_with_rank_and_parent( "Hylidae", Taxon::FAMILY, @Anura )
+      set_taxon_with_rank_and_parent( "Pseudacris", Taxon::GENUS, @Hylidae )
+      set_taxon_with_rank_and_parent( "Pseudacris regilla", Taxon::SPECIES, @Pseudacris )
 
-    Taxon.reset_iconic_taxa_constants_for_tests
+      set_taxon_with_rank_and_parent( "Aves", Taxon::CLASS, @Chordata, is_iconic: true )
+      set_taxon_with_rank_and_parent( "Apodiformes", Taxon::ORDER, @Aves )
+      set_taxon_with_rank_and_parent( "Trochilidae", Taxon::FAMILY, @Apodiformes )
+      set_taxon_with_rank_and_parent( "Calypte", Taxon::GENUS, @Trochilidae )
+      set_taxon_with_rank_and_parent( "Calypte anna", Taxon::SPECIES, @Calypte, common_name: "Anna's Hummingbird" )
+
+      set_taxon_with_rank_and_parent( "Plantae", Taxon::KINGDOM, @Life, is_iconic: true )
+      set_taxon_with_rank_and_parent( "Magnoliophyta", Taxon::PHYLUM, @Plantae )
+      set_taxon_with_rank_and_parent( "Magnoliopsida", Taxon::CLASS, @Magnoliophyta )
+      set_taxon_with_rank_and_parent( "Myrtales", Taxon::ORDER, @Magnoliopsida )
+      set_taxon_with_rank_and_parent( "Onagraceae", Taxon::FAMILY, @Myrtales )
+      set_taxon_with_rank_and_parent( "Clarkia", Taxon::GENUS, @Onagraceae )
+      set_taxon_with_rank_and_parent( "Clarkia amoena", Taxon::SPECIES, @Clarkia )
```

Comment:
Earlier `Taxon::GENUS` rank was used here; after my changes validation error occurred complaining that "rank level must be finer than parent". I believe that `Taxon::SPECIES` is a correct one here so made the corresponding change. Didn't investigate why the validation error didn't show up earlier when parent was set during update, not create.